frontend/benefit/handler/src/components/messenger/Messages.tsx

```diff
@@ -0,0 +1,51 @@
+import { MESSAGE_TYPES } from 'benefit-shared/constants';
+import { Message } from 'benefit-shared/types/application';
+import camelCase from 'lodash/camelCase';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import MessageComponent from 'shared/components/messaging/Message';
+import { $MessagesList } from 'shared/components/messaging/Messaging.sc';
+import { MessageVariant } from 'shared/types/messages';
+
+interface ComponentProps {
+  data: Message[];
+  variant: MessageVariant;
+  withScroll?: boolean;
+}
+
+const Messages: React.FC<ComponentProps> = ({ data, variant, withScroll }) => {
+  const { t } = useTranslation();
+  const scrollMessagesRef = React.createRef<HTMLDivElement>();
+
+  React.useEffect(() => {
+    scrollMessagesRef?.current?.scrollIntoView({ behavior: 'smooth' });
+  }, [data, scrollMessagesRef]);
+
+  return (
+    <$MessagesList variant={variant}>
+      <>
```

Comment:
`<>` is not needed because `<$MessagesList>` is parent?

Reply:
fixed